Tests/BackendTest/StudentServiceTest.cs

```diff
@@ -20,6 +21,32 @@ public StudentServiceTest(Mongo2GoFixture mongoFixture)
             _mongoFixture = mongoFixture;
         }
 
+        
+        [Fact]
+        public async Task AssingTimetableToStudent()
+        {
+            IMongoDatabase database = _mongoFixture.MongoClient.GetDatabase("StudentsDB");
+            StudyGroupService grpsrvc = new StudyGroupService(database);
+            CourseService crsrv = new CourseService(database);
+            ISchoolScheduleProxy proxy = new FakeProxy();
+
+            StudyGroup sg = await grpsrvc.GetStudyGroupAsync("5ZI001", crsrv, proxy);
+            Student s = new Student();
+            s.Timetable = sg.Timetable.Clone();
+            s.StudyGroupId = sg.Id;
+
+            var newBlock = new Block();
+
+            sg.Timetable.AddNewBlock(newBlock);
+
+            s.Timetable.GetBlock(newBlock.Id).Should().BeNull();
+
+            var newBlockSt = new Block();
+            s.Timetable.AddNewBlock(newBlockSt);
+
+            sg.Timetable.GetBlock(newBlockSt.Id).Should().BeNull();
+        }
+
```

Comment:
This will be problem, @davidm1766 just removed id from block in #65

Reply:
OK I fixed It